apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -0,0 +1,796 @@
+        <target>Как использовать SimpleX Chat</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="How to use markdown" xml:space="preserve">
+        <source>How to use markdown</source>
+        <target>Как форматировать</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="If you cannot meet in person, you can **scan QR code in the video call**, or your contact can share an invitation link." xml:space="preserve">
+        <source>If you cannot meet in person, you can **scan QR code in the video call**, or your contact can share an invitation link.</source>
+        <target>Если вы не можете встретиться лично, вы можете **сосканировать QR код во время видео-звонка**, или ваш контакт может отправить вам ссылку.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="If you cannot meet in person, you can **show QR code in the video call**, or you can share the invitation link via any other channel." xml:space="preserve">
+        <source>If you cannot meet in person, you can **show QR code in the video call**, or you can share the invitation link via any other channel.</source>
+        <target>Если вы не можете встретиться лично, вы можете **показать QR код во время видео-звонка** или отправить ссылку через любой другой канал связи.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="If you received SimpleX Chat invitation link you can open it in your browser:" xml:space="preserve">
+        <source>If you received SimpleX Chat invitation link you can open it in your browser:</source>
+        <target>Если Вы получили ссылку с приглашением из SimpleX Chat, Вы можете открыть её в браузере:</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Install [SimpleX Chat for terminal](https://github.com/simplex-chat/simplex-chat)" xml:space="preserve">
+        <source>Install [SimpleX Chat for terminal](https://github.com/simplex-chat/simplex-chat)</source>
+        <target>[SimpleX Chat для терминала](https://github.com/simplex-chat/simplex-chat)</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Invalid connection link" xml:space="preserve">
+        <source>Invalid connection link</source>
+        <target>Ошибка в ссылке контакта</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Make sure SMP server addresses are in correct format, line separated and are not duplicated." xml:space="preserve">
+        <source>Make sure SMP server addresses are in correct format, line separated and are not duplicated.</source>
+        <target>Пожалуйста, проверьте, что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется.</target>
```

Comment:
```suggestion
        <target>Пожалуйста проверьте что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется.</target>
```

Reply:
I understand lack of punctuation is incorrect but it reads simpler and less official

Reply:
🤷‍♂️